textdb/textdb-perftest/src/main/java/edu/uci/ics/textdb/perftest/excelsink/ExcelSinkTest.java

```diff
@@ -0,0 +1,134 @@
+package edu.uci.ics.textdb.perftest.excelsink;
```

Comment:
@diaojinggang A general comment about the test case: since you already setup the testing framework, it will be good to add more test cases with the same framework but different examples to cover more cases, especially corner cases.